wagtail/utils/setup.py

```diff
@@ -29,10 +29,11 @@ def bump_client_version(self):
         Writes the current Wagtail version number into package.json
         """
         path = os.path.join(".", "client", "package.json")
-        input_file = open(path)
 
+        package = {}
```

Comment:
Nit: there's no need to define `package` first - since the `except` terminates the app